src/extension.ts

```diff
@@ -25,33 +25,41 @@ export function start(context: theia.PluginContext): void {
     mocha.useColors(true);
 
     const e = (c: any) => console.log(c);
-
     ncp(context.extensionPath, '/projects/Che-Java-Tests', (err: any) => {
         if (err) {
             return console.error(err);
         }
+        const getSrcDocPath = path.resolve('/projects/Che-Java-Tests/testWorkspace/src/main/java/org/my/sample', 'MyHelloText.java');
+        const getSrcDocUri = theia.Uri.file(getSrcDocPath);
+        theia.commands.executeCommand('file-search.openFile', getSrcDocUri).then(() => {
```

Comment:
should vscode.workspace.openTextDocument be better ?

Reply:
replaced with `theia.workspace.openTextDocument`